cmd/prepare-storage.go

```diff
@@ -185,7 +185,7 @@ func connectLoadInitFormats(retryCount int, firstDisk bool, endpoints Endpoints,
 
 	for i, err := range errs {
 		if err != nil {
-			if err == errDiskNotFound && retryCount >= 10 {
+			if err == errDiskNotFound && retryCount >= 9 {
```

Comment:
Can you replace retryCount parameter with verboseLogging as boolean ?

Reply:
Hi, vadmeste, thx for your comment. I added a new commit, not sure is the right way, would pls take a look?